File: src/services/Services.NestJS/assistant-options.ts
export const ASSISTANT_OPTIONS = 'ASSISTANT_OPTIONS';

export interface AssistantOptions {
  systemPrompt: string;
}

export function createAssistantOptions(): AssistantOptions {
  return {
    systemPrompt: process.env.ASSISTANT_SYSTEM_PROMPT || 'You are a helpful meal planning assistant.',
  };
}

File: src/services/Services.DotNet/AssistantOptions.cs
namespace Services.DotNet;

public sealed class AssistantOptions
{
    public string SystemPrompt { get; set; } = "You are a helpful meal planning assistant.";

    public int MaximumToolCallsPerRequest { get; set; } = 10;

    public int MaximumConversationSize { get; set; } = 100;

    public TimeSpan MaximumExecutionTime { get; set; } = TimeSpan.FromMinutes(2);
}